quarkus/extensions/kogito-quarkus-rules-extension/kogito-quarkus-rules-integration-test-session/src/test/java/org/kie/kogito/quarkus/drools/RuntimeIT.java

```diff
@@ -15,14 +15,23 @@
  */
 package org.kie.kogito.quarkus.drools;
 
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
 import javax.inject.Inject;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.kie.api.KieBase;
+import org.kie.api.definition.KiePackage;
 import org.kie.api.runtime.KieSession;
+import org.kie.api.time.SessionPseudoClock;
 import org.kie.kogito.legacy.rules.KieRuntimeBuilder;
 
 import io.quarkus.test.junit.QuarkusTest;
 
+import static io.smallrye.common.constraint.Assert.assertTrue;
```

Comment:
Fix import and use junit :)
```suggestion
import static org.junit.jupiter.api.Assertions.assertTrue;
```